WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,4 +1,7 @@
-- Font size and line height are now supported for block themes.
-- Add new stat cards in the Settings menu on the Stats screen.
-- When changing the media in an Image block that’s also a post’s Featured Image, get a prompt to replace the Featured Image.
-- Share text, images, and video in the app using your device’s Share action.
+* [*] Comment editor: Notification comments support in the new editor [https://github.com/wordpress-mobile/WordPress-Android/pull/15884]
+* [*] Site creation: Fixed bug where sites created within the app were not given the correct time zone, leading to post scheduling issues. [https://github.com/wordpress-mobile/WordPress-Android/pull/15904]
+* [*] Login Epilogue: Fixed bug where if no sites available, then new login epilogue screen without clickable sites and with "create new site" is shown instead of no sites empty "My Site" view. [https://github.com/wordpress-mobile/WordPress-Android/pull/15944]
+* [*] Fixes flickering when changing the preview mode in Page or Site design picker [https://github.com/wordpress-mobile/WordPress-Android/pull/15943]
+* [**] Jetpack App: Signup using WordPress.com, create your favorite site and enjoy the Reader inside Jetpack app. [https://github.com/wordpress-mobile/WordPress-Android/pull/15946]
+* [*] Block Editor: Stop editor from unexpectedly closing while composing a story block when memory is low [https://github.com/wordpress-mobile/WordPress-Android/pull/15960]
+* [*] Improves the error message shown when trying to create a new site with non-English characters in the domain name [https://github.com/wordpress-mobile/WordPress-Android/pull/15969]
```

Comment:
Doubt: Is site creation applicable to Jetpack? 🤔

Reply:
Should we pick a different wording for those entries for Jetpack then? Because it seems strange that an existing user of Jetpack, who never saw the flow for creating a new site before since it wasn't enabled in 19.2, would see "improve an error message on… [something that was not an existing feature before]" 🤔  wdyt?

Reply:
Yeah, I feel the same. Maybe we should remove `Improves the error...`  entry for the JP app.

Reply:
Addressed in [681333f](https://github.com/wordpress-mobile/WordPress-Android/pull/15986/commits/681333ff5bee95e216234470184fd4363154d507)